src/components/shared/btn-reset.tsx

```
@@ -0,0 +1,27 @@
+import { X } from "lucide-react";
+import { Button } from '@components/ui/button';
+import { cn } from '@lib/utils/cn.utils';
+import { useTranslations } from "next-intl";
+
+type ResetButtonProps = {
+  onClick: () => void;
+  label?: string;
+  className?: string;
+}
+
+export default function BtnReset({ onClick, label = "Reset", className }: ResetButtonProps) {
```

ESLint

[error] 'label' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)